src/sortable.h

```diff
@@ -35,11 +35,14 @@ typedef struct RSSortingVector {
 /* RSSortingTable defines the length and names of the fields in a sorting vector. It is saved as
  * part of the spec */
 typedef struct {
-  uint8_t len;
-  struct sortField {
-    const char *name;
-    RSValueType type;
-  } fields[RS_SORTABLES_MAX];
+  const char *name;
+  RSValueType type;
+} RSSortField;
+
+typedef struct {
+  uint16_t len;
+  uint16_t cap;
```

Comment:
Do we use `len` for anything but iteration of the fields in `RSSortingTable_GetFieldIdx`? Maybe t's possible to "null terminate" the list and stay just with `cap`?

Reply:
@rafie I think this is easier to read, and since both are `uint16_t`, it doesn't take any additional due to padding.